apps/frontend/src/pages/contributors/libs/components/is-hidden-header/is-hidden-header.tsx

```diff
@@ -0,0 +1,11 @@
+import { Icon } from "~/libs/components/components.js";
+
+import styles from "./styles.module.css";
+
+const IsHiddenHeader = (): JSX.Element => (
+	<div className={styles["header-with-icon"]}>
+		Is Hidden <Icon height={15} name="circleQuestion" width={15} />
```

Comment:
In the design, this icon is meant to be with a tooltip, but if we rename the column on UI to Do Not Track, then I think we can remove this icon at all, especially taking into consideration that it's described inside edit modal